libyul/AsmJsonConverter.cpp

```diff
@@ -75,6 +76,20 @@ Json AsmJsonConverter::operator()(Literal const& _node) const
 	return ret;
 }
 
+Json AsmJsonConverter::operator()(Verbatim const& _node) const
+{
+	Json ret = createAstNode(originLocationOf(_node), nativeLocationOf(_node), "YulIdentifier");
```

Comment:
I don't see what is unusual about this.

Reply:
Ah, you mean that it is all treated as `YulIdentifier`?

Reply:
> I don't see what is unusual about this.

That proves the point I guess :)

Yeah, it's the fact that we're pretending this is an identifier rather than properly exposing the node in the AST. It may be easy to answer in the context of the PR, but when viewing the code you usually don't have that context and things like this should have a comment.

Reply:
Yeah that is fair enough. I have added a comment that it is to avoid a breaking change in the json interface